DMCompiler/Optimizer/PeepholeOptimizations.cs

```diff
@@ -116,6 +116,34 @@ public void Apply(DMCompiler compiler, List<IAnnotatedBytecode> input, int index
     }
 }
 
+// PushReferenceValue [ref]
+// PushString [string]
+// DereferenceIndex [value] [string]
```

Comment:
DereferenceIndex doesn't have any arguments
```suggestion
// DereferenceIndex
```